OptimizelySDKShared.podspec

```diff
@@ -16,6 +16,7 @@ Pod::Spec.new do |s|
                               "OptimizelySDKShared/OptimizelySDKShared/OPTLYDatabaseEntity.{h,m}"
   s.public_header_files     = "OptimizelySDKShared/OptimizelySDKShared/*.h"
   s.framework               = "Foundation"
+  s.library                 = "sqlite3"
```

Comment:
I would like to propose that this new *.podspec line be changed to
    s.ios.library = "sqlite3"
since Optimizely's objective-c-sdk's support for tvOS does not include OPTLYFMDB nor sqlite3 .